config/fusor-installer.answers.yaml

```diff
@@ -38,7 +38,12 @@ capsule:
 
 foreman::plugin::bootdisk: true
 foreman::plugin::discovery:
+  # We do not want to seek the images from the internet.
+  # want them installed from RPM, foreman-discovery-image
   install_images: false
```

Comment:
Recommend that we remove lines 41-43 (comment and install_images).  I believe the default behavior is not to include the images.